Corona-Warn-App/src/androidTest/java/testhelpers/FragmentTestModuleRegistrar.kt

```diff
@@ -119,6 +122,9 @@ import de.rki.coronawarnapp.ui.vaccination.CovidCertificateInfoFragmentTestModul
         RATProfileQrCodeFragmentTestModule::class,
 
         QrCodeScannerFragmentTestModule::class,
+
+        // --------- Settings ---------
+        NotificationSettingsFragmentModule::class,
```

Comment:
Please call it `NotificationSettingsFragmentTestModule` ? to distinguish it from actual module